server.js

```diff
@@ -389,6 +389,15 @@ app.use(/^\/pl\/course_instance\/[0-9]+\/instructor\/effectiveUser(\/?.*)$/, fun
     }
 });
 
+// Redirect Instruct question page to Student Version if we don't have Instructor authz.
+app.use('/pl/course_instance/:course_instance_id/instructor/assessment/:assessment_id/questions', function(req, res, next) {
+    if (!res.locals.authz_data.has_instructor_view) {
+        res.redirect(`/pl/course_instance/${req.params.course_instance_id}/assessment/${req.params.assessment_id}/`);
+    } else {
+        next();
+    }
+});
+
```

Comment:
Should we move this into the existing question route?

https://github.com/PrairieLearn/PrairieLearn/blob/e08ce66b68fd458f9663df590b74909f5d408b78/server.js#L442-L445

Reply:
But on line 402 it's going to check whether the user has permission to view the `/pl/course_instance/:course_instance_id/instructor` first and it will redirect to the 403 error page if the user doesn't. So I don't think it will work if we move it down?

Reply:
Ah ok, gotcha